lib/models/device.coffee

```diff
@@ -855,7 +862,7 @@ getDeviceModel = (deps, opts) ->
 	exports.stopApplication = (uuidOrId, callback) ->
 		exports.get(uuidOrId).then (device) ->
 			ensureSupervisorCompatibility(device.supervisor_version, MIN_SUPERVISOR_APPS_API).then ->
-				appId = device.application[0].id
+				appId = device.belongs_to__application[0].id
```

Comment:
Like above
> I think that this needs to change since we are not expanding the `belongs_to__application`.
I guess that we should either expand it or use `device.belongs_to__application.__id`.